vars/cloudFoundryDeploy.groovy

```diff
@@ -412,17 +413,29 @@ private deploy(def cfApiStatement, def cfDeployStatement, def config, Closure po
         usernameVariable: 'username'
     )]) {
 
+        def postDeployCommand = config.postDeployCommand
+
+        if(postDeployCommand) {
+            echo "[INFO][${STEP_NAME}] Will Execute Post Deploy Command: '${postDeployCommand}'."
+        }
+        else {
+            echo "[INFO][${STEP_NAME}] No Post Deploy Command to Execute."
+        }
+
         def cfTraceFile = 'cf.log'
 
         def deployScript = """#!/bin/bash
             set +x
             set -e
             export HOME=${config.dockerWorkspace}
             export CF_TRACE=${cfTraceFile}
+            export CF_STAGING_TIMEOUT=60
```

Comment:
do we really want hardcoded values here?

Reply:
Agreed.  Just testing it make sure it was actually affecting my long deploy before adding a param to pass in this value.